setup.cfg

```diff
@@ -1,6 +1,6 @@
 [flake8]
 show-source = True
-ignore = E731
+ignore = E731, W503
```

Comment:
A lot of these changes seem to be completely unrelated to Python 3, but change the style of the binary operators to be after the line instead of before. That's why you shut off this warning. Can we please focus this PR on Python 3 support and not unrelated coding style changes?

In the future I want to move the code style to black so we don't have this discussion anymore.

Reply:
The problem is that either W503 or W504 is triggering and W504 seems to be the correct one.

https://lintlyci.github.io/Flake8Rules/rules/W503.html
https://lintlyci.github.io/Flake8Rules/rules/W504.html

Reply:
All right, I withdraw my objection and we'll worry about it again when we have black.